composer.json

```
@@ -27,7 +27,15 @@
         "rector/rector": "0.13.9"
     },
     "suggest": {
-        "ext-fileinfo": "Improves mime type detection for files"
+        "ext-imagick": "If you use ImageMagickHandler",
+        "ext-simplexml": "If you format XML",
+        "ext-mysqli": "If you use MySQL",
+        "ext-oci8": "If you use Oracle Database",
+        "ext-pgsql": "If you use PostgreSQL",
+        "ext-sqlsrv": "If you use SQL Server",
+        "ext-sqlite3": "If you use SQLite3",
+        "ext-fileinfo": "Improves mime type detection for files",
+        "ext-readline": "Improves CLI::input() usability"
```

I think Composer Normalize will alphabetize these:
```suggestion
        "ext-fileinfo": "Improves mime type detection for files",
        "ext-imagick": "If you use ImageMagickHandler",
        "ext-mysqli": "If you use MySQL",
        "ext-oci8": "If you use Oracle Database",
        "ext-pgsql": "If you use PostgreSQL",
        "ext-readline": "Improves CLI::input() usability",
        "ext-simplexml": "If you format XML",
        "ext-sqlite3": "If you use SQLite3",
        "ext-sqlsrv": "If you use SQL Server"
```

---

What is *Composer Normalize* ?

---

It's a Composer plugin. `ergebnis/composer-normalize`

---

It seems we don't use it.

```console
$ composer normalize
                                       
  Command "normalize" is not defined.  
```

---

Oops! My mistake. I use it locally and it is part of DevKit so I assumed we had it here, but maybe not. We can always come back and normalize later - merging this now so it makes it into `4.2.2`.